packages/block-library/src/latest-posts/index.php

```diff
@@ -89,37 +125,49 @@ function register_block_core_latest_posts() {
 		'core/latest-posts',
 		array(
 			'attributes'      => array(
-				'align'           => array(
+				'align'                   => array(
 					'type' => 'string',
 					'enum' => array( 'left', 'center', 'right', 'wide', 'full' ),
 				),
-				'className'       => array(
+				'className'               => array(
 					'type' => 'string',
 				),
-				'categories'      => array(
+				'categories'              => array(
 					'type' => 'string',
 				),
-				'postsToShow'     => array(
+				'postCount'               => array(
```

Comment:
@draganescu Isn't this change going to cause headaches for existing sites that have populated `postsToShow` (both in PHP and JS)? Doesn't there need to be some migration of the old `postsToShow` prop to the new `postShow` prop?

Reply:
Good catch @westonruter. I think you are right, it should be updated to the old name to avoid all the dance with deprecations.